internal/olm/operator/bundle/install.go

```diff
@@ -107,79 +108,76 @@ func (i *Install) setup(ctx context.Context) error {
 
 	var declcfg *declarativeconfig.DeclarativeConfig
 
-	directoryName := filepath.Join("/tmp", strings.Split(csv.Name, ".")[0]+"-index")
-	fileName := filepath.Join(directoryName, "testFBC")
-	bundleChannel := strings.Split(labels[registrybundle.ChannelsLabel], ",")[0]
-
 	catalogLabels, err := registryutil.GetImageLabels(ctx, nil, i.IndexImageCatalogCreator.IndexImage, false)
 	if err != nil {
 		return fmt.Errorf("get index image labels: %v", err)
 	}
 
+	directoryName := filepath.Join("/tmp", strings.Split(csv.Name, ".")[0]+"-index")
+	fileName := filepath.Join(directoryName, "testFBC")
+
 	if _, hasDBLabel := catalogLabels[containertools.DbLocationLabel]; hasDBLabel {
-		log.Infof("Converting SQLite Image to a File-Based Catalog")
+		i.IndexImageCatalogCreator.HasDBLabel = true
```

Comment:
The other label field will _always_ be false, right? Go will default it to that iirc.

Reply:
Setting to `i.IndexImageCatalogCreator.HasDBLabel = true` when fbc label doesn't exist on the image. This should probably cover the default case and all cases where an image won't have either of those labels.

Reply:
Based on the most recent commit's name, should we error out in the `else` block?